src/baldr/edgeinfo.cc

```diff
@@ -415,24 +460,44 @@ int8_t EdgeInfo::layer() const {
   return static_cast<int8_t>(value.front());
 }
 
-std::string EdgeInfo::level() const {
+bool EdgeInfo::includes_level(float lvl) {
```

Comment:
I have removed the `level()` method and replaced it with `includes_level(lvl)`, because that's all we really do with the levels information at this point, and just returning the levels array that includes the separator values is of little use.